models/renalStructures_UNEST_segmentation/requirements.txt

```diff
@@ -0,0 +1,5 @@
+nibabel
+timm
+ignite
+fire
```

Comment:
Hi @tangy5 , this file is not mentioned in `README.md`, and all libs are mentioned in `metadata.json`, thus it may not needed I think.

Reply:
Thanks! I've removed the requirements.txt file.